test/spec/electrode.spec.js

```diff
@@ -92,108 +92,102 @@ describe("electrode-server", function() {
             logLevel
           }
         });
-      } catch (e) {
-        error = e;
-      }
-      expect(error, "expected error thrown").to.exist;
-      if (!_.includes(error.message, "is already in use")) {
-        throw error;
+      }),
+      error => {
+        expect(error).to.be.an("Error");
+        expect(error.message).includes("is already in use");
       }
-    } finally {
-      await stopServer(server);
-    }
+    );
   });
 
-  it("should fail for listener errors", async function() {
-    let error;
-    try {
-      await electrodeServer({}, require("../decor/decor3"));
-    } catch (e) {
-      error = e;
-    }
-    expect(error, "expected error thrown").to.exist;
-    expect(error.message).includes("test listner error");
+  it("should fail for listener errors", () => {
+    return asyncVerify(
+      expectError(() => electrodeServer({}, require("../decor/decor3"))),
+      error => {
+        expect(error).to.be.an("Error");
+        expect(error.message).includes("test listner error");
+      }
+    );
   });
 
-  it("should fail for listener errors from decor array with func", async function() {
-    let error;
-    try {
-      await electrodeServer({}, [require("../decor/decor4")]);
```

Comment:
this seems more readable than using asyncVerify

Reply:
ok, that's debatable, but readability is not the reason asyncVerify's created or used.